test/dotnet-new.IntegrationTests/DotnetNewTestTemplatesTests.cs

```diff
@@ -0,0 +1,259 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.DotNet.Cli.New.IntegrationTests
+{
+    public class DotnetNewTestTemplatesTests : BaseIntegrationTest
+    {
+        private readonly ITestOutputHelper _log;
+
+        private static readonly ImmutableArray<string> SupportedTargetFrameworks =
+        [
+            "net10.0",
```

Comment:
Did you want to target any downlevel versions? In net11, would these tests target net11 and net10 or be updated to just net11? If you're only ever going to target current, you might be able to reuse the values in ToolsetInfo.cs so as to reduce the work needed when upgrading TFMs.